Yosemite/YosemiteTests/Stores/CustomerStoreTests.swift

```diff
@@ -48,6 +48,50 @@ final class CustomerStoreTests: XCTestCase {
         super.tearDown()
     }
 
+    // MARK: - CustomerAction.createCustomer
+
+    func test_createCustomer_persists_customer() throws {
```

Comment:
Same here: it would be better to move these tests after the tests for `synchronizeAllCustomers`.

Reply:
Updated order in 5c8f176.